### PR TITLE
Used open graph protocol to add site meta data

### PR DESCRIPTION
**Pull request by SirChronicle**
parameters used:

title: "Null NEU"
description: "A cybersecurity club at Northeastern University focused on learning and sharing knowledge in security."

### CODE REVIEW

File: layouts/partials/opengraph.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{ with .Site.Params.title }}
+<meta property="og:title" content="{{ . }}" />
+{{ end }}
```
Comment (Parthiv-M) [inline layouts/partials/opengraph.html:3]: Can be simplified to `<meta property="og:title" content="{{ .Site.Params.title }}" />`

File: layouts/partials/opengraph.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+{{ with .Site.Params.description }}
+<meta property="og:description" content="{{ . }}" />
+{{ end }}
```
Comment (Parthiv-M) [inline layouts/partials/opengraph.html:6]: Same as above

**Review by Parthiv-M — CHANGES_REQUESTED** [review]
The approach is spot on, just some more improvements!

- Would be helpful if we can add the following tags

  ```
  <meta name="description" content="{{ if .Description }}{{ .Description }}{{ else }}{{ .Site.Params.description }}{{ end }}" />
  
  <meta name="twitter:title" content="{{ .Title }} | {{ .Site.Title }}" />
  <meta name="twitter:image" content="{{ .Site.Params.og_image | absURL }}" />
  
  <meta itemprop="name" content="{{ .Title }} | {{ .Site.Title }}" />
  <meta name="application-name" content="{{ .Title }} | {{ .Site.Title }}" />
  
  <meta property="og:title" content="{{ .Title }} | {{ .Site.Title }}" />
  <meta property="og:type" content="website" />
  <meta property="og:title" content="{{ .Site.Title }}" />
  <meta property="og:description"content="{{ if .Description }}{{ .Description }}{{ else }}{{ .Site.Params.description }}{{ end }}" />
  <meta property="og:site_name" content="{{ .Site.Title }}" />
  <meta property="og:url" content="{{ .Permalink }}" />
  <meta property="og:locale" content="en">
  <meta property="og:image" content="{{ .Params.Site.BaseURL }}{{ .Site.Params.og_image | relURL }}" />
  <meta property="og:image:secure_url" content="{{ .Site.Params.og_image | absURL }}" />
  <meta property="og:type" content="website" />
  ```

This is just something I got off another resource, please validate and modify accordingly before including in our partial.

**Comment by SirChronicle** [comment]
I've added the suggested meta tags after reviewing each (commented all tags for images for future updates) and updated the relevant files. Let me know if there's anything else to adjust!

**Pull request merged**